index.bs

```diff
@@ -3279,6 +3321,50 @@ interface AudioParam {
 Attributes</h4>
 
 <dl dfn-type=attribute dfn-for="AudioParam">
+	: <dfn>automationRate</dfn>
+	::
+		The automation rate for the {{AudioParam}}.  The
+		default value depends on the actual {{AudioParam}};
+		see the description of the {{AudioParam}} for the
+		default value.
+
+		Some nodes have additional <dfn dfn>automation rate constraints</dfn> as follows:
+
+		: {{AudioListener}}
```

Comment:
Is this the right place to mention these? Shouldn't we describe them in each node's section?

Reply:
I was only following what we already have for other constraints:  https://webaudio.github.io/web-audio-api/#audionode-channelcount-constraints, https://webaudio.github.io/web-audio-api/#audionode-channelcountmode-constraints

Reply:
Oh, each `AudioParam` table does mention if there are constraints on the selectable rate and links to here.